bigtable/use-cases/fraudDetection/terraform/main.tf

```diff
@@ -0,0 +1,163 @@
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.23.0"
```

Comment:
is this version specifically needed? could consider setting it as `hashicorp/google" version = ">= 4.23.0"` to futureproof, but defer to you